Add basic README

JonShort opened the pull request:
- Added basic feature list
- Added basic usage information
- Added basic environment requirements
- Added basic installation instructions

JonShort committed:
Added basic README

JonShort committed:
Updated README with extra information

JonShort committed:
Missed bullets

JulijanasJezov commented on README.md:
```diff
@@ -0,0 +1,35 @@
+- install (manual) [https://www.java.com/en/download/help/download_options.xml](https://www.java.com/en/download/help/download_options.xml)
+
+**Gradle**
+- check current version `gradle -v`
```
can just run gradlew build and that will download gradle and build the project

JonShort replied:
oh nice, will update

JonShort committed:
- Made gradlew executable on unix
- Updated version of gradle used in gradlew
- Updated readme to reflect no need to install gradle
